python/downgrades/503c0516fba2e5da9570f00eb34ef43025ecb8fb/py_exprs.ql

```diff
@@ -26,7 +26,12 @@
  */
 bindingset[new_index]
 int old_index(int new_index) {
-  if new_index < 18 then result = new_index else result + (19 - 18) = new_index
+  if new_index < 18
+  then result = new_index
+  else
+    if new_index >= 19
+    then result + (19 - 18) = new_index
+    else none()
```

Comment:
Code scanning raises a valid point, but could we not just rewrite this as follows?
```ql
new_index < 18 and result = new_index
or
new_index >= 19 and result + (19-18) = new_index
```
In fact, would it be more obvious what's going on if we instead wrote `new_index > 18` in the second disjunct? It took me a moment to realise that `18` is excluded.

Reply:
I would like to use this as a template. If, instead of 18 and 19, it would have been, say, 12 and 19, the guards should be `new_index < 12` and `new_index >= 19`. So I would like to keep to your current suggestion, and perhaps add some comments.